server/cluster/metadata/types.go

```diff
@@ -107,15 +107,16 @@ type CloseTableRequest struct {
 }
 
 type MigrateTableRequest struct {
-	SchemaName string
-	TableNames []string
-	OldShardID storage.ShardID
-	NewShardID storage.ShardID
+	SchemaName            string
+	TableNames            []string
+	OldShardID            storage.ShardID
+	latestOldShardVersion uint64
```

Comment:
Shall we introduce a struct call VersionShard to simplify these four fields into two?

Reply:
In fact, `MigrateTableRequest` is useless now,  the migration table is not currently in actual use. I will add a todo here and modify it together with the subsequent reconstruction of the migration table.